common/web/types/build.sh

```diff
@@ -100,7 +100,10 @@ function do_build() {
 function do_test() {
   eslint .
   tsc --build test
-  c8 --skip-full --reporter=lcov --reporter=text mocha "${builder_extra_params[@]}"
+  readonly C8_THRESHOLD=75
+  c8 -skip-full --reporter=lcov --reporter=text --lines $C8_THRESHOLD --statements $C8_THRESHOLD --branches $C8_THRESHOLD --functions $C8_THRESHOLD mocha "${builder_extra_params[@]}"
```

Comment:
`--skip-full` appears to have lost a hyphen here.  Is that intended?

Reply:
```suggestion
  c8 --skip-full --reporter=lcov --reporter=text --lines $C8_THRESHOLD --statements $C8_THRESHOLD --branches $C8_THRESHOLD --functions $C8_THRESHOLD mocha "${builder_extra_params[@]}"
```